<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>corewindowform</class>
 <widget class="QWidget" name="corewindowform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="NavListView" name="navListView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>201</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>1021</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="DeviceStatusForm" name="DeviceStatusPage"/>
   <widget class="QWidget" name="FaultDealingPage"/>
   <widget class="QWidget" name="page_4"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="CancelpushButton">
   <property name="geometry">
    <rect>
     <x>992</x>
     <y>597</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavListView</class>
   <extends>QListView</extends>
   <header>navlistview.h</header>
  </customwidget>
  <customwidget>
   <class>DeviceStatusForm</class>
   <extends>QWidget</extends>
   <header>devicestatusform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
